app/controllers/cohorts/client_notes_controller.rb

```diff
@@ -29,6 +29,14 @@ def create
         ),
       )
       @cohort_client.touch
+      # send notifications
+      if note_params[:send_notification].present? && note_params[:recipients].present?
+        token = Token.tokenize(client_notes_path(client_id: @cohort_client.client_id))
+        note_params[:recipients].reject(&:blank?).map(&:to_i).each do |id|
+          user = User.find(id)
+          TokenMailer.note_added(user, token).deliver_later if user.present?
+        end
+      end
```

Comment:
Does the TokenMailer generate a link that doesn't require a login?  If it doesn't, we should switch this and the other note controller to use a standard mailer.

Reply:
No, the `TokensController` just looks up the token and redirects to the path stored in the DB. It looks like it is just a URL shortener/ away to obscure any data in the URL from emails.